Jenkinsfile

```diff
@@ -79,37 +61,16 @@ def buildDetails = { vars ->
 }
 
 def notify = { ->
-  if (env.BRANCH_NAME == devBranch ||
-      env.BRANCH_NAME == stagingBranch ||
-      env.BRANCH_NAME == prodBranch) {
+  if (env.BRANCH_NAME == DEV_BRANCH ||
+      env.BRANCH_NAME == STAGING_BRANCH ||
+      env.BRANCH_NAME == PROD_BRANCH) {
     message = "vets-website ${env.BRANCH_NAME} branch CI failed. |${env.RUN_DISPLAY_URL}".stripMargin()
     slackSend message: message,
     color: 'danger',
     failOnError: true
   }
 }
 
-def comment_broken_links = {
-  // Pull down console log replacing URL with IP since we can't hit internal DNS
-  sh "curl -O \$(echo ${env.BUILD_URL} | sed 's/jenkins.vetsgov-internal/172.31.1.100/')consoleText"
-
-  // Find all lines with broken links in production build
-  def broken_links = sh (
-    script: 'grep -o \'\\[production\\].*>>> href: ".*",\' consoleText',
-    returnStdout: true
-  ).trim()
-
-  def github = GitHub.connect()
-  def repo = github.getRepository('department-of-veterans-affairs/vets-website')
-  def pr = repo.queryPullRequests().head("department-of-veterans-affairs:${env.BRANCH_NAME}").list().asList().get(0)
-
-
-  // Post our comment with broken links formatted as a Markdown table
-  pr.comment("### Broken links found by Jenkins\n\n|File| Link URL to be fixed|\n|--|--|\n" +
-             broken_links.replaceAll(/\[production\] |>>> href: |,/,"|") +
-             "\n\n _Note: Long file names or URLs may be cut-off_")
-}
-
```

Comment:
This could go, because page are edited over at `vagov-content` so the link-checker belongs there instead.

Reply:
Do we have links in layout files? We might miss a broken link if that's the case.